src/Microsoft.TestPlatform.ObjectModel/Constants.cs

```diff
@@ -183,7 +183,12 @@ public static class Constants
         /// <summary>
         /// Default results directory.
         /// </summary>
-        public static readonly string DefaultResultsDirectory = Path.Combine(Directory.GetCurrentDirectory(), ResultsDirectoryName);
+        public static readonly string DefaultResultsDirectory =
+#if NETSTANDARD1_0
```

Comment:
I am just curious what is the logic behind `if` statements? Why to use `.` if it is NETStandart1.0?

Reply:
NETStandard 1.0 doesn't support file IO, and we can't get `Directory.GetCurrentDirectory()`; we are using `.` instead of current directory there.